qa/sc_evm_bootstrap.py

```diff
@@ -214,6 +214,16 @@ def run_test(self):
         # check if header contains correct gasUsed (2 * eoa to eoa transfer gas costs)
         assert_equal(21000*2, sc_best_block['block']['header']['gasUsed'])
 
+        # test eth_estimateGas with ERC20 (TestERC20.sol with additional mint function) contract bytecode
+        request = {
```

Comment:
looks to be a good candidate to be put into the separate py test.
We need to extend it and check the gas estimation for:
1) EOA to EOA
2) EOA to EOA with NON empty data
3) Forger stake delegation/withdrawal
4) SC to MC withdrawal
5) other common smart contracts (any regression data from geth?)
6) eth